src/helpers/transport/query.rs

```diff
@@ -120,7 +120,7 @@ impl Substep for QueryType {}
 #[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
 pub struct IpaQueryConfig {
     pub per_user_credit_cap: u32,
-    pub max_breakdown_key: u128,
+    pub max_breakdown_key: u32,
```

Comment:
I'm not sure why you changed all of these from u128 to u32. Doesn't seem related to the title and summary of this PR. What was the motivation for this?

Reply:
This is to just align with other parameter types. I initially used `u128` to make conversions between `usize` easier, but it's clearly unnecessarily large for breakdown keys.